Service/Capture.php

```diff
@@ -255,6 +255,17 @@ public function createOrder(string $rvvupId, Quote $quote, string $origin): Vali
                 $quote->getReservedOrderId(),
                 $origin
             );
+
+            if ($this->sdkProxy->isOrderVoidable($rvvupId)) {
+                $rvvupPaymentId = $payment->getAdditionalInformation(Method::PAYMENT_ID);
+                $reason = 'INVALID_DATA';
+                if ($e->getMessage() == 'Some of the products are out of stock.') {
```

Comment:
I am getting INVALID DATA when out of stock because the message is:
 ```There are no source items with the in stock status\nSome of the products are out of stock.```

Should we just do a message contains "out of stock"?

Reply:
Great spot!
Changed that